test-configs.yaml

```diff
@@ -442,6 +442,11 @@ device_types:
     name: 'hi6220-hikey'
     mach: hisilicon
     class: arm64-dtb
+
+  hsdk:
+    name: 'hsdk'
```

Comment:
The `name` attribute is not necessary when it's the same as the name of the YAML entry.

Reply:
Still same comment.  It's a benign issue, but it's not necessary so it makes maintenance a bit easier to drop this `name` attribute.